lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -388,6 +394,33 @@ def configure(conf)
       @resource.labels.freeze
       @common_labels.freeze
 
+      # If prometheus monitoring is enabled, register metrics in the default
+      # registry and store metric objects for future use. If metrics are
+      # registed already, just store the metric objects.
+      if @prometheus_monitoring_enabled
```

Comment:
Let's also fix the rubocop complaints (the build [failed](https://travis-ci.org/GoogleCloudPlatform/fluent-plugin-google-cloud/jobs/248830189)).